### PR TITLE
Propagate current path through the rust code

### PR DESCRIPTION
**Pull request by arduano**
Propagate current path through the rust code

Propagating current path (workdir) through the rust code.

### CODE REVIEW

File: src/eval/execution.rs [hunk 1]
```diff
@@ -172,16 +171,11 @@ fn nix_value_from_module(
     scope: &mut v8::ContextScope<v8::HandleScope>,
     nix_value: v8::Local<v8::Function>,
```
Comment (urbas) [inline src/eval/execution.rs:172]: Not related to your change, but I believe this variable has a misleading name. I think this should be `nix_module_fn` or something like that. If you agree, would you mind changing it while you're here?

Comment (arduano) [reply src/eval/execution.rs:172]: Changed

File: src/tests/mod.rs [hunk 2]
```diff
@@ -15,17 +15,19 @@ mod literals;
 mod operators;
 
 fn eval_ok(nix_expr: &str) -> Value {
-    match evaluate(nix_expr) {
+    let workdir = std::env::current_dir().unwrap();
+    match evaluate(nix_expr, &workdir) {
         Ok(val) => val,
         Err(err) => panic!(
-            "eval '{}' shouldn't fail.\nError message: {}",
+            "eval '{}' shouldn't fail.\nError message: {:?}",
```
Comment (urbas) [inline src/tests/mod.rs:22]: Total nitpick (feel free to ignore or do in a separate PR):
```suggestion
            "eval '{nix_expr}' shouldn't fail.\nError message: {err:?}",
```
Then the line below can be removed.

File: src/tests/mod.rs [hunk 3]
```diff
@@ -15,17 +15,19 @@ mod literals;
...
             nix_expr, err
         ),
     }
 }
 
 fn eval_err(nix_expr: &str) -> NixErrorKind {
-    evaluate(nix_expr)
+    let workdir = std::env::current_dir().unwrap();
+    evaluate(nix_expr, &workdir)
         .expect_err(&format!("eval '{}' expected an error", nix_expr))
```
Comment (urbas) [inline src/tests/mod.rs:31]: Same as above (feel free to ignore etc.):
```suggestion
        .expect_err(&format!("eval '{}' expected an error", nix_expr))
```
```suggestion
        .expect_err(&format!("eval '{nix_expr}' expected an error"))
```

**Review by urbas — DISMISSED** [review]

File: src/tests/mod.rs (hunk 3 above)
Comment (arduano) [reply src/tests/mod.rs:31]: If you want these things to be picked up automatically, we can add clippy

**Review by urbas — APPROVED** [review]

**Pull request merged**